scripts/host-backup-restore/host-backup

```diff
@@ -23,11 +29,19 @@ tar --exclude 'tmp/*' --exclude 'tmp/.*' --exclude 'var/crash/*' --exclude 'var/
     --exclude 'var/log/audit*' \
     --exclude 'var/xsconfig/*' --exclude 'var/xsconfig/.*' \
     --exclude 'var/xapi/[0-9a-f]*-[0-9a-f]*-[0-9a-f]*-[0-9a-f]*-[0-9a-f]*' \
+    --exclude-from ${EXCLUDE_FILE} --exclude ${EXCLUDE_FILE} \
```

Comment:
Why not just use the `--warning='no-file-ignored'` option to declutter the output, rather than running find and then having to clean up the `EXCLUDE` files?